wgsl/index.bs

```diff
@@ -585,20 +561,15 @@ variable_decoration
   </xmp>
 </div>
 
-<pre class='def'>
-builtin_decoration
-  : POSITION
-  | VERTEX_IDX
-  | INSTANCE_IDX
-  | FRONT_FACING
-  | FRAG_COORD
-  | FRAG_DEPTH
-  | NUM_WORKGROUPS
-  | WORKGROUP_SIZE
-  | LOCAL_INVOC_ID
-  | LOCAL_INVOC_IDX
-  | GLOBAL_INVOC_ID
```

Comment:
The list of builtin decorations seems to have gone missing?

Reply:
It was actually redundant, and listed below. (I added text indicating this now)